packages/gatsby-transformer-remark/src/extend-node-type.js

```diff
@@ -453,11 +452,11 @@ module.exports = (
             const amountToPruneLastNode =
               pruneLength - (unprunedExcerpt.length - lastTextNode.value.length)
             if (!truncate) {
-              lastTextNode.value = prune(
-                lastTextNode.value,
-                amountToPruneLastNode,
-                `…`
-              )
+              lastTextNode.value = _.truncate(lastTextNode.value, {
+                length: amountToPruneLastNode,
+                separator: /,?\.* +/,
```

Comment:
Is this the behavior of `prune`, or why exactly do we need this? Also let's be explicit with the space, e.g. `\s` 

Also a note that I generally like to encapsulate this as a var if we're going to re-use it a few times, e.g. if we hoist to near the top of the file like

```js
const EXCERPT_SEPARATOR = /,?\.*\s+/
```

Reply:
Right - but this could theoretically be a breaking change then, right?

Reply:
Without the separator, it definitely was a breaking change. With it in place, I tried to mimic it to existing prune implementation. Existing snapshot tests are passing with this. There might be some boundary conditions that I might have missed, can't be 100% sure about that.